Update/some_011.txt

```diff
@@ -2238,7 +2238,7 @@ void main()
 //r田舎町の出身のおかげで、この街に来てから血のにじむような辛苦を味わった。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "田舎町の出身のおかげで、この街に来てから血のにじむような辛苦を味わった。",
-			NULL, "I've suffered bloody hardship since I came to this city, thanks to the rural town I came from.", Line_Normal);
+			NULL, "I've suffered excrutiating hardship since I came to this city, thanks to being born and raised in the countryside.", Line_Normal);
```

Comment:
```suggestion
			NULL, "I've suffered excruciating hardship since I came to this city, thanks to being born and raised in the countryside.", Line_Normal);
```